packages/core/package.json

```diff
@@ -107,8 +107,8 @@
     }
   },
   "dependencies": {
-    "@wagmi/chains": "workspace:*",
-    "@wagmi/connectors": "workspace:*",
+    "@wagmi/chains": "workspace:../../references/packages/chains",
+    "@wagmi/connectors": "workspace:../../references/packages/connectors",
```

Comment:
pnpm throws the following error when running `pnpm i`. my guess is it does validation before running the `preinstall` script.

```bash
 ERR_PNPM_NO_MATCHING_VERSION_INSIDE_WORKSPACE  In packages/core: No matching version found for @wagmi/chains@* inside the workspace
```

Switching to [relative path references](https://pnpm.io/workspaces#referencing-workspace-packages-through-their-relative-path) makes everything work smoothly.